apps/ecs/index.js

```diff
@@ -162,10 +162,26 @@ module.exports = {
       next: '/worker-details'
     },
     '/worker-details': {
+      behaviours: [checkValidation],
+      fields: [
+        'worker-been-in-uk-before-1988-full-name',
+        'worker-been-in-uk-before-1988-dob',
+        'worker-been-in-uk-before-1988-nationality'
+      ],
       next: '/reference-number'
     },
     '/reference-number': {
-      next: '/worker-address-uk'
+      fields: ['worker-reference-number'],
+      forks: [
```

Comment:
for conditions with binary values, I would suggest using one of them as the default next step, this will reduce the amount of checking